content/open-source/wolfi/building-a-wolfi-package.md

```diff
@@ -192,7 +192,7 @@ For example, this is how a step in the subpackages section would be written, usi
       - uses: split/dev
 ```
 
-**Looping with Ranges**
+### Looping with Ranges
```

Comment:
I can't comment in the `subpackages` above, but it would be probably good to also add something about serviceable packages, which is another reason for having a subpackage. I'll send some pointers in case you want to do that.

Reply:
Cool, that sounds good. Can you send me some reference about it, I would like to learn more!

Reply:
Ok, I read the internal wiki page about it. It feels like an advanced topic that won't be simple to explain in a couple paragraphs. If we had a public documentation resource that we could link out, it would be fine to add a note, but in this case it is more complex than what the article was initially designed for. We can create a ticket to write this public-facing piece and update this guide once we have that resource.

Reply:
That sounds good, thanks!